docker-compose.unit-tests.yml

```diff
@@ -7,11 +7,12 @@ services:
       args:
         GITHUB_TOKEN: ${GITHUB_TOKEN}
     volumes:
-      - ./artifacts/tests/docker-output/:/src/artifacts/tests/output/
+      - ./output:/output
     environment:
       NPM_READ_TOKEN: ${NPM_READ_TOKEN}
     command: >-
       /bin/sh -c "
       dotnet build --nologo --no-restore --configuration Debug &&
-      dotnet run --project build --configuration Release --no-build --no-launch-profile -- --configuration Debug unit-tests
+      dotnet run --project build --configuration Release --no-build --no-launch-profile -- --configuration Debug unit-tests &&
+      cp -R /src/tests/. /output
```

Comment:
why do we copy test sources to output?

Reply:
we copy all to be able to use wildcard in a github action 
```
path: './output/**/*.trx
```

Reply:
aren't they saved to artifacts folder?

Reply:
saved where?

Reply:
where?

Reply:
@evgeniy-ac @FrolYo did you figure out what has to be done here?